shared/src/main/scala/minitest/runner/Runner.scala

```diff
@@ -36,8 +36,8 @@ final class Runner(
     None
   }
 
-  def serializeTask(task: BaseTask, serializer: TaskDef => String): String =
-    serializer(task.taskDef())
+  def serializeTask(task: BaseTask, serializer: TaskDef | Null => String): String =
```

Comment:
I cannot find any documentation or source code for `sbt.testing.Task` and `sbt.testing.TaskDef`